modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/js/CKEditor5ReactClassicEditor.tsx

```diff
@@ -3,18 +3,18 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {EditorConfig} from 'ckeditor5';
 import {
 	CKEditor5ClassicEditor as ClassicEditor,
-	advancedClassicEditorConfig,
+	ClassicEditorConfig,
+	EClassicEditorConfigPreset,
 } from 'frontend-editor-ckeditor-web';
 import React from 'react';
 
 const CKEditor5ReactClassicEditor = () => {
-	const config: EditorConfig = {
-		...advancedClassicEditorConfig,
+	const config: ClassicEditorConfig = {
 		initialData:
 			'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Nunc id cursus metus aliquam eleifend mi in nulla. Quam adipiscing vitae proin sagittis nisl rhoncus. Suspendisse faucibus interdum posuere lorem. Nullam ac tortor vitae purus faucibus ornare. Ac felis donec et odio pellentesque diam. Nulla at volutpat diam ut. Posuere urna nec tincidunt praesent semper feugiat nibh. Gravida quis blandit turpis cursus. Proin libero nunc consequat interdum varius. Sollicitudin ac orci phasellus egestas tellus rutrum tellus pellentesque. Neque volutpat ac tincidunt vitae semper quis lectus nulla at. Odio euismod lacinia at quis risus sed vulputate odio ut. Augue lacus viverra vitae congue eu consequat ac. Elementum sagittis vitae et leo duis ut diam. Diam quis enim lobortis scelerisque fermentum dui faucibus.',
+		preset: EClassicEditorConfigPreset.BASIC,
```

Comment:
I switched basic preset to React sample, and advanced to Classic sample. This is to be able to have image selection in the sample. Image selection requires `filebrowserImageBrowseUrl`, and this is set in in `EditorConfigContributor`. React sample cannot easily apply `EditorConfigContributor`.